Dining/models.py

```diff
@@ -192,6 +192,14 @@ def external_dining_entries(self):
         """All dining entries that are not for external people."""
         return DiningEntryExternal.objects.filter(dining_list=self)
 
+    def can_modify(self, user):
+        """Users can modify when the dining list is open, owner can modify when dining list is still adjustable."""
```

Comment:
Should we not also take the purchaser into account here?

Reply:
Oh yes thanks